### DISCUSSION TITLE
Support sortedmulti() in multi-sig wallet descriptors

### DISCUSSION THREAD

**Discussion by eliasnaur** [btcpayserver/btcpayserver#4569] [Ideas / Feature Requests]
Sparrow generates multi-sig descriptor in the pattern:

wsh(sortedmulti(m,[<mfp>/48h/0h/0h/2h]xpub.../<0;1>/*,[<mfp>/48h/0h/0h/2h]xpub.../<0;1>/*,[<mfp>/48h/0h/0h/2h]xpub.../<0;1>/*))#<checksum>

However BTCPay 1.7.4 doesn't accept them. It returns the error "Invalid wallet format: Unexpected 's'".

Changing `sortedmulti` to `multi` generates the error: "Extended public key ... Invalid wallet format: CheckSum mismatch. Expected: <checksum>; Actual: <a different checksum> ".

Replacing `sortedmulti` and removing the #<checksum> results in "Invalid wallet format: Unexpected 'm'".

**Comment by dennisreimann** [comment]
I dealt with that just yesterday and actually it's the Sparrow export that contains the `<0;1>` as a [yet not standardized proposal](https://github.com/bitcoin/bitcoin/pull/22838). It's a combination of the Receive and Change descriptor. 

We will support it once that proposal gets accepted – for now you can make it work by replacing that part with simply `0` and removing the checksum.
